dotCMS/src/main/java/com/dotmarketing/portlets/workflows/model/WorkflowAction.java

```diff
@@ -226,6 +227,16 @@ public boolean hasPushPublishActionlet() {
 		return this.pushPublishActionlet;
 	}
 
+
+	/**
+	 * Returns true if the action has at least one action only batch
+	 * @return Boolean true if the action is only batch
+	 */
+
+	public boolean hasOnlyBatchActionlet() {
```

Comment:
Maybe the method should be named `isOnlyBatchActionlet()`

Reply:
Name looks correcto to me.